lib/index.js

```
@@ -234,7 +234,12 @@ Appboy.prototype.group = function(group) {
   var userId = group.userId();
   var groupIdKey = 'ab_segment_group_' + group.groupId();
 
-  window.appboy.changeUser(userId);
+  try {
+    window.appboy.changeUser(userId);
+  } catch (e) {
+    // Ignore the error.
```

Can we bubble the error to the ajs metrics Prateek set up?